azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -603,6 +608,17 @@ def _check_processes_in_agent_cgroup(self):
                 self._report_agent_cgroups_procs(agent_cgroup_proc_names, unexpected)
                 raise CGroupsException("The agent's cgroup includes unexpected processes: {0}".format(unexpected))
 
+        def get_logcollector_unit_properties(self):
+            """
+            Returns the systemd unit properties for the log collector process
+            """
+            logcollector_properties = ["--property=CPUAccounting=yes", "--property=MemoryAccounting=yes", "--property=CPUQuota={0}".format(LOGCOLLECTOR_CPU_QUOTA)]
```

Comment:
Yes, we set CPUQuota limit in logcollector slice. I left the common properties in the slice

Reply:
why do we set here?

Reply:
If we don't set the properties explicitly here, then some strange behaviors may happen. For example, CPUAccounting=yes in the slice but if I don't set it here then the collect-logs.scope directory is not created in V1. Also CPUQuota is set in the slice, but if I don't set it here then the collect-logs.scope directory does not have a cpu.max file. 

When I tested, we got the expected behavior when all properties were explicit here

Reply:
let's add a comment in the code so that people do not change this inadvertently

looks to me like systemd is making those decisions (if the scope needs to be tracked, creates a new cgroup, otherwise uses the cgroup for the slice) -- this is just me speculating, though

Reply:
Looks to me we need to set explicitly for scope in order to track scope. Do we need for slice? If it required, please add a comment. Thanks